docs/pages/Banner.md

```diff
@@ -56,7 +56,7 @@ Uses the color from `theme.palette.primary.base`.
 <Banner
   p={2}
   color="error"
-  iconName="Warning"
+  icon={<Icon name="Warning"/>}
```

Comment:
Should we be passing in more specific icons like `icon={<Warning />}`

Reply:
That's what I would think, but apparently those aren't available however the MDX is configured. Any ideas?

Reply:
That makes sense. Maybe we should leave this till the move to storybook docs then.

Reply:
afaik you have to wait for publish on this. @unscsprt is correct.